### PR TITLE
3d mymaps: readonly mode

### PR DESCRIPTION
**Pull request by tonio**
Based on #1959

### CODE REVIEW

File: geoportailv3/static/js/maincontroller.js [hunk 1]
```diff
@@ -407,7 +407,9 @@ app.MainController.prototype.enable3dCallback_ = function(active) {
   if (!active) {
     return;
   }
-  this['mymapsOpen'] = false;
+  if (!goog.isDef(this.ngeoLocation_.getParam('map_id'))) {
```
Comment (gberaudo) [inline geoportailv3/static/js/maincontroller.js]: I think there exists an `hasParam` method in ngeo.
When the param exists it means we are not in edition?

File: geoportailv3/static/js/mymaps/mymapsdirective.js [hunk 2]
```diff
@@ -135,6 +135,12 @@ app.MymapsDirectiveController = function($scope, $compile, $sce,
    */
   this.map_ = this['map'];
 
+  /**
+   * @const {?app.olcs.Lux3DManager}
+   * @export
```
Comment (gberaudo) [inline geoportailv3/static/js/mymaps/mymapsdirective.js]: Do you need it exported? If not it should be private.

**Comment by tonio** [comment]
There are a few case that I can't test on my instance, merging now to be able to test them on dev instance.

**Comment by gberaudo** [comment]
OK

**Pull request merged**